packages/transition-backend/src/models/db/migrations/20231206083200_createTransferableNodeTbl.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
+import { Knex } from 'knex';
+
+const tableName = 'tr_transit_node_transferable';
+
+export async function up(knex: Knex): Promise<unknown> {
+    return knex.schema.createTable(tableName, (table: Knex.TableBuilder) => {
+        table.uuid('origin_node_id').notNullable().index(); // required
```

Comment:
Possible to add mode here, since later on, we could use bicycle or even car as transfer mode

Reply:
Do we need to add it now, or when we actually plan to use it ? Since we don't really know what it will look like, it's harder to do if it's the right design

Reply:
Indeed, there's a lot to change to support various modes. We'll do that later

Reply:
PLease add an issue :-)

Reply:
#159 already covers this case